docs/source/release/v3.10.0/indirect_inelastic.rst

```diff
@@ -17,6 +17,11 @@ Algorithms
 - :ref:`BASISReduction311 <algm-BASISReduction311>` has been deprecated (2017-03-11). Use :ref:`BASISReduction <algm-BASISReduction>` instead.
 - :ref:`BASISReduction <algm-BASISReduction>` includes now an option to compute and save the dynamic susceptibility.
 
+QuickRuns
+~~~~~~~~~
+
+- :ref:`IndirectDiffScan <algm-IndirectDiffScan>` to to improve diffraction reduction workflow
```

Comment:
Double `to` and missing dot. Maybe explain a bit how this improves the workflow?